### DISCUSSION TITLE
Visualisations croisant conso et démographie

### DISCUSSION THREAD

**Discussion by inesdartiguenave** [MTES-MCT/sparte#426] [Idées]
### Les nouvelles données d'évolutions démographiques sont là :
https://www.insee.fr/fr/statistiques/3698339

avec la géographie communale au 01/01/2022, comme les données du Cerema

### et celle des ménages ici :
https://www.insee.fr/fr/statistiques/5359146

Les nouvelles données de population vont jusqu'en 2021 (2019 actuellement), celles des ménages jusqu'en 2020 (2018 actuellement)

**Comment by Phil-Loriot** [comment]
Il faudrait changer le type de diagramme pour en avoir un semblable à celui-ci (à maquetter pour choisir les options) 

[Graphique conso_pop.odg](https://github.com/user-attachments/files/15739324/Graphique.conso_pop.odg)

**Comment by alexisig** [comment]
#219
